verifier/src/lib.rs

```diff
@@ -23,6 +23,13 @@ pub enum VerificationError {
         /// The actual attributes that were present
         actual: Attributes,
     },
+    /// The config id did not match expected:{expected} actual:{actual}
```

Comment:
I'm curious on thoughts for moving errors to an error.rs module.
and the and/or verifiers to a dedicated module, can't think of a good name right now.

The main reason is that I don't like how lib.rs pulls in report_body and report_body.rs is depending on contents directly from lib.rs.  Although I suppose it's not much different with lib.rs declaring the modules and other modules in the crate depending on them from being declared in lib.rs